packages/squiggle-lang/__tests__/ReducerProject/ReducerProject_tutorial_1_multisource_test.res

```diff
@@ -0,0 +1,112 @@
+@@warning("-44")
+module InternalExpressionValue = ReducerInterface_InternalExpressionValue
+module Project = ForTS_ReducerProject
+module Bindings = Reducer_Bindings
+
+open Jest
+open Expect
+open Expect.Operators
+
+describe("ReducerProject Tutorial", () => {
+  describe("Multi source", () => {
+    /*
+     Case "Running multiple sources" */
+    test("Chaining", () => {
+      let project = Project.createProject()
+      /* This time let's add 3 sources and chain them together */
+      Project.setSource(project, "source1", "x=1")
+
+      Project.setSource(project, "source2", "y=2")
+      /* To run, source2 depends on source1 */
+      Project.setContinues(project, "source2", ["source1"])
```

Comment:
I find the API of ``setSource`` and ``setContinues`` confusing. Do we really need these to be two different function calls? Can we just have one, like, 

``setSource(~project, ~name, ~contents, ~dependencyNames)``

Or similar?